lib/pymedphys/_mosaiq/sessions.py

```diff
@@ -0,0 +1,186 @@
+# session and session offset calculator
+
+from datetime import datetime, timedelta
+
+from pymedphys._imports import sklearn
+
+from pymedphys import mosaiq
```

Comment:
This might cause a circular import down the line. Normally I make sure to call the most public API from the tests, but internally within PyMedPhys I usually call the internal APIs